### ISSUE TITLE
Broken svg sizing when using inline styles

### ISSUE THREAD

**Issue by dgudim** [KosmosisDire/obsidian-webpage-export#230]
When exporting a document with inline svg which have inline styles, they are broken in the export 

In obsidian: 
```
<svg width="800" height="800" preserveAspectRatio="xMidYMid" version="1.1" viewBox="-38.4 -81.966 221.87 491.8" xmlns="http://www.w3.org/2000/svg" style="width:150px;">
```

In the export:

```
<svg height="1024" viewBox="0 0 1024 1024" fill="none" xmlns="http://www.w3.org/2000/svg" style="width:1024px;max-width:100%">
```

The width is set to 1024 for some reason

**Comment by KosmosisDire** [comment]
Interesting, thanks for the report. I will look into this.

**Comment by KosmosisDire** [comment]
This should be fixed in 1.8.0. Let me know if it still does not work as expected and reopen the issue. If you find any new issues please submit a new issue :)
Thanks!
